components/CommonCompAlert.js

```diff
@@ -2,10 +2,10 @@ import { Alert, Platform } from 'react-native'
 
 const alertPolyfill = (title, description, options, extra) => {
     const result = window.confirm([title, description].filter(Boolean).join('\n'))
-
+    console.log(result)
```

Comment:
Do we want to keep this in here, or was it for testing?

Reply:
yeah this is for testing purpose so that in case it fails, we know it is from here